Model/ResourceModel/Reservation/Collection/Grid.php

```diff
@@ -32,10 +31,11 @@ public function __construct(
         FetchStrategy $fetchStrategy,
         EventManager $eventManager,
         $mainTable = 'allegro_reservations',
-        $resourceModel = Reservation::class,
+        $resourceModel = null,
         $identifierName = null,
         $connectionName = null
     ) {
+        $resourceModel = Reservation::class;
```

Comment:
Is this intentional? This change override whatever you pass as resourcemodel argument.

Reply:
I did it to resolve "Possible useless method overriding detected" warning. What's the best solution for this warning?

Reply:
did you check what is the rule behind "Possible useless method overriding detected" warning?

Reply:
Done